source/includes/fundamentals/code-snippets/builders/AggBuilders.java

```diff
@@ -302,4 +302,13 @@ private void matchStage() {
         match(eq("title", "The Shawshank Redemption"));
         // end match
     }
+
+    private void documents() {
+        // begin documents
+        documents(asList(
+                new Document("title", "The Shawshank Redemption"),
+                new Document("title", "Back to the Future"),
+                new Document("title", "Jurassic Park")));
```

Comment:
Is this an acceptable syntax for this stage? One of my teammates noticed that it varies from what is used in the [unit tests.](https://github.com/mongodb/mongo-java-driver/blob/a19fad2b3d93ee818132f14377e42cfaa268e8ee/driver-core/src/test/functional/com/mongodb/client/model/AggregatesTest.java#L170-L171)

Reply:
Yes.

While this variant can be more verbose than the "parse" variant, I think we want to avoid encouraging the use of parse (it will be mildly slower, and has other caveats that are incidentally unimportant for our unit tests).